src/test/java/pl/piekoszek/gorskimatches/equation/EquationGeneratorTest.java

```diff
@@ -0,0 +1,12 @@
+package pl.piekoszek.gorskimatches.equation;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class EquationGeneratorTest {
```

Comment:
Hmmm, looks like this test doesn't do much

Reply:
Yep, I can delete it as I prepared it but didn't do much with it :D